roles/config-rh-sso/tasks/main.yml

```diff
@@ -7,3 +7,7 @@
   loop: "{{ realms }}"
   loop_control:
     loop_var: realm
+- include_tasks: 'manage-clients.yml'
+  loop: "{{ clients }}"
```

Comment:
Can we add a `defaults([])` to avoid bailing if no clients are defined? I.e.: `{{ clients | default([]) }}`

Reply:
... unless you want to bail in that event?

Reply:
Fixed